build_tools/travis/install.sh

```diff
@@ -37,7 +37,6 @@ then
              -O miniconda.sh
     fi
     chmod +x miniconda.sh && ./miniconda.sh -b -f
-    conda update --yes conda
```

Comment:
Why do you remove this here?

Reply:
There seems to be an error with the build. I'm not sure what changed but after installing conda and trying to update the packages, it will say the package already exists/cannot be installed.
```
UnsatisfiableError: The following specifications were found to be in conflict
- cffi
```

Reply:
But that's a general CI error then? Why is it part of this specifically? If so, we should make that a separate PR so it's easier to revert.